src/main/java/de/ellpeck/actuallyadditions/mod/items/ItemLeafBlower.java

```diff
@@ -107,7 +108,8 @@ public boolean breakStuff(World world, int x, int y, int z){
                     BlockPos pos = new BlockPos(x+reachX, y+reachY, z+reachZ);
                     Block block = world.getBlockState(pos).getBlock();
 
-                    if(block != null && ((block instanceof BlockBush || block instanceof IShearable) && (this.isAdvanced || !block.isLeaves(world.getBlockState(pos), world, pos)))){
+                    if((block instanceof BlockBush || block instanceof IShearable) &&
+                       (this.isAdvanced || !block.isLeaves(world.getBlockState(pos), world, pos))){
```

Comment:
Seriously

Reply:
About moving imports on top and other formatting features. It'll better, if you post somewhere imported formatting configuration from IntelliJ IDEA (because most of the stuff occurs 'on the fly').